.github/workflows/ci.yml

```diff
@@ -4,11 +4,11 @@ on:
   pull_request: {}
 jobs:
   run:
-    runs-on: macOS-latest
+    runs-on: macos-11
     name: Xcode ${{ matrix.xcode }}
     strategy:
       matrix:
-        xcode: ["12"]
+        xcode: ["12.5.1", "13.0"]
```

Comment:
Do we still need 12.5.1? @yonaskolb  I think it is only takes long time to build

Reply:
I think it's wise to leave both 12.5.x and 13.0, at least for a few weeks until Xcode 13 adoption is mainstream. It's a nice guard against accidentally regressing one of the major Xcode builds out there. And the builds run in parallel, so it doesn't actually take longer AFAICT.

I'd suggest revisiting and removing 12.x in a couple months.

Reply:
Agree, let's keep till there's at least a 13.1